webrender/src/tiling.rs

```diff
@@ -1682,20 +1702,21 @@ pub struct StackingContext {
     composite_ops: CompositeOps,
     clip_source: ClipSource,
     clip_cache_info: Option<MaskCacheInfo>,
+    packed_layer_index: PackedLayerIndex,
 }
 
 #[derive(Debug, Clone)]
 #[repr(C)]
-pub struct PackedStackingContext {
+pub struct PackedLayer {
     transform: LayerToWorldTransform,
     inv_transform: WorldToLayerTransform,
     local_clip_rect: LayerRect,
     screen_vertices: [WorldPoint4D; 4],
 }
 
-impl Default for PackedStackingContext {
-    fn default() -> PackedStackingContext {
-        PackedStackingContext {
+impl Default for PackedLayer {
```

Comment:
Since there was already `impl Default for PackedStackingContext`, this is not something that has to be done in your PR. But for the future I think we should remove the `impl Default` and just have the same thing as an `fn empty() -> PackedLayer` constructor. No implementation change needed.